Gemfile.lock

```diff
@@ -8,21 +8,112 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
+    actioncable (5.0.7.2)
```

Comment:
Why are there so many updates to the gemlock file but no updates to the gemfile?

Reply:
I think I covered your answer here https://github.com/learn-co/railway_ipc_gem/pull/4#discussion_r358494897